api/v1/controllers/subjects.js

```diff
@@ -60,6 +60,29 @@ function checkDuplicates(tagsArr) {
 }
 
 /**
+ * Validates the correct filter of Subject Tags
+ * passed in query parameters
+ * @param {Array} subjectTagsArray Subject Tags Array
+ */
+function subjectTagsValidation(subjectTagsArray) {
```

Comment:
Would be more appropriate to use a verb for the function name here: "validateSubjectTags"